runtimes/alibi-explain/mlserver_alibi_explain/explainers/white_box_runtime.py

```diff
@@ -29,17 +29,23 @@ def __init__(self, settings: ModelSettings, explainer_class: Type[Explainer]):
         super().__init__(settings, explainer_settings)
 
     async def load(self) -> bool:
+        # white box explainers requires access to the full inference model
         self._inference_model = await self._get_inference_model()
 
         if self.alibi_explain_settings.init_parameters is not None:
+            # Instantiate explainer with init parameters (and give it full inference model)
             init_parameters = self.alibi_explain_settings.init_parameters
-            # white box explainers requires access to the inference model
-            init_parameters["model"] = self._inference_model
-            self._model = self._explainer_class(**init_parameters)  # type: ignore
+            self._model = self._explainer_class(self._inference_model, **init_parameters)  # type: ignore
```

Comment:
Passing `self._inference_model` as unnamed arg instead of kwarg, because some explainers name this arg differently (i.e. `predict_fn`, `model` or `predictor`).